DataFormats/Reconstruction/src/TrackParametrizationWithError.cxx

```diff
@@ -754,11 +755,17 @@ GPUd() auto TrackParametrizationWithError<value_T>::getPredictedChi2Quiet(const
   return (d * (szz * d - sdz * z) + z * (sdd * z - d * sdz)) / det;
 }
 
-#if !defined(GPUCA_GPUCODE) && !defined(GPUCA_STANDALONE) // Disable function relying on ROOT SMatrix on GPU
```

Comment:
maybe this needs to be kept for standalone build @davidrohr ?

Reply:
well, it should not call ROOT stuff. If you got rid of all ROOT code, then you can also remove the #ifndef GPUCA_STANDALONE. Anyway, worst case I'll fix it later.

Reply:
I removed any `GPUCA_STANDALONE` reference as they wee just guarding ROOT stuff.